.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,92 @@
+      options:
+      - label: I definitively want to report a bug
+        required: true
+  - type: checkboxes
+    attributes:
+      label: Is there already an existing issue for this?
+      description: Please search this repository to see if an issue already exists for the bug you encountered.
+      options:
+      - label: I have searched the existing issues
+        required: true
+  - type: textarea
+    attributes:
+      label: Describe the bug
+      description: |
+        Describe the issue you are experiencing here to communicate to the maintainers. Tell us what you were trying to do and what happened. Help us understand the issue by providing valuable context.
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: Steps to reproduce the behavior
+      description: |
+        If the problem is reproducable, list the steps here. If the problem can't be reproduced and is sporadic, please provide some details on how often and when the issue happens.
+      placeholder: |
+          1. Go to '...'
+          2. Click on '....'
+          3. Scroll down to '....'
+          4. Observed error
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: Expected behavior
+      description: If applicable, describe what you expected to happen.
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: Screenshots
+      description: If applicable, add screenshots to help explain the problem.
+    validations:
+      required: false
+  - type: textarea
+    attributes:
+      label: Environment
+      description: |
+        The deCONZ and firmware versions are found in:
+        - Phoscon App: Menu -> Settings -> Gateway, OR
+        - deCONZ GUI: Menu > Help > About deCONZ.
+      value: |
+           - Host system: (Raspberry Pi / PC / NAS / MAC)
+           - Running method: (Raspbian / Ubuntu / Home Assistent deCONZ Add-on / Marthoc Docker container / Windows / Virtual Machine)
+           - Firmware version: (26xxyy00)
+           - deCONZ version: (2.xx.yy)
+           - Device: (ConBee I / ConBee II / RaspBee I / RaspBee II)
+           - Do you use an USB extension cable: (yes / no) -- only relevant for ConBee I/II
+           - Is there any other USB or serial devices connected to the host system? If so: Which?
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: deCONZ Logs
+      description: |
+        Debug logs can help to investigate certain problems like device pairing and communication issues. Logging can be enabled on the command line as described in: https://github.com/dresden-elektronik/deconz-rest-plugin/wiki/deCONZ-debug-flags
+        Secondly, for versions above 2.05.87: In the deCONZ GUI: click Help > debug view. Output of APS,ZDP, ZCL and INFO can be handy.
```

Comment:
I generally like to see:
APS
APS_L2
ERROR
ERROR_L2
INFO
INFO_L2

I am not sure on ZCL / ZDP.

Reply:
Agree. Made it a bit more cautious though.